packages/x-charts-pro/src/context/ChartDataProviderPro/ChartDataProviderPro.tsx

```diff
@@ -63,321 +63,4 @@ function ChartDataProviderPro(props: ChartDataProviderProProps) {
   );
 }
 
-ChartDataProviderPro.propTypes = {
```

Comment:
This is dead code, `ChartDataProviderPro` is a private API so shouldn't have prop types. It also happens to be wrong, go to https://next.mui.com/x/react-charts/heatmap/ and you will get a runtime warning with the `width` prop required.